backends/gstreamer/src/player.rs

```diff
@@ -47,6 +47,8 @@ fn metadata_from_media_info(media_info: &PlayerMediaInfo) -> Result<Metadata, ()
         .get_container_format()
         .unwrap_or_else(|| "".to_owned());
 
+    let seekable = media_info.is_seekable();
```

Comment:
As this flag is highly dependent on if the user set ``set_stream_type()``, but is value is, if the use set Seekable{Fast}, to know if the media container demuxer  is capable of seeking. But in theory we should control the installed demuxers and which are usable by us.

So, I'm not really sure if this metadata is required.